.travis.yml

```diff
@@ -26,11 +26,13 @@ matrix:
       env: SOLR_VERSION="4.10.3" TEST_CONFIG="phpunit-integration-legacy-solr.xml" SOLR_CONFS="eZ/Publish/Core/Search/Solr/Content/Resources/schema.xml"
 # 5.5
     - php: 5.5
-      env: TEST_CONFIG="phpunit.xml" SYMFONY_VERSION="~2.7@dev"
+      env: TEST_CONFIG="phpunit.xml" SYMFONY_VERSION="~2.7"
     - php: 5.5
       env: TEST_CONFIG="phpunit-integration-legacy.xml" DB="mysql" DATABASE="mysql://root@localhost/$DB_NAME"
     - php: 5.5
       env: ELASTICSEARCH_VERSION="1.4.2" TEST_CONFIG="phpunit-integration-legacy-elasticsearch.xml"
+    - php: 5.5
+      env: TEST_CONFIG="phpunit.xml" SYMFONY_VERSION="~2.8@dev"
```

Comment:
As composer is bumped to use 2.7 by default we should rather change any use of ` SYMFONY_VERSION="~2.7"` to ` SYMFONY_VERSION="~2.8@dev"`, no need to add additional job given 2.3 is out.